workflow/pred_temperature_autoencoder.ipynb

```diff
@@ -0,0 +1,1298 @@
+{
```

Comment:
<p>I am not sure what you mean with "but only the encoding part"</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/AI4S2S/cookbook/pull/7/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='28'/>

Reply:
<p>Thanks for noticing. In this notebook, I only use the encoder (left) part of the model, without encoding and attention for outputs (right). This is also a structure used in a famous model called BERT. But I agree that this sounds confusing. I rephrase this part and add a new figure to it. </p>